plugins/obs-filters/noise-suppress-filter.c

```diff
@@ -340,8 +340,6 @@ static inline void alloc_channel(struct noise_suppress_data *ng,
 #ifdef LIBSPEEXDSP_ENABLED
 	ng->spx_states[channel] =
 		speex_preprocess_state_init((int)frames, sample_rate);
-#else
-	UNUSED_PARAMETER(sample_rate);
```

Comment:
The related function definition should not have `OBS_UNUSED` since it is used if speexdsp is enabled. So it would be better to remove the `OBS_UNUSED` and keep the `UNUSED_PARAMETER.`

Reply:
Don't forget to mark this "review" as resolved.